modules/claims_api/app/swagger/claims_api/v2/production/swagger.json

```diff
@@ -3886,7 +3876,11 @@
                                 "approximateBeginDate": "2020-06",
                                 "approximateEndDate": "2020-07"
                               }
-                            ]
+                            ],
+                            "federalActivation": {
+                              "activationDate": "2024-08-26",
+                              "anticipatedSeparationDate": "2024-10-03"
+                            }
```

Comment:
Should we remove `federalActivation` here?

Reply:
We still want to support federalActivation as a data object. And we still want to map that data to Form526. But we only expect that federalActivation object to be included for BDD claims. Whereas, our maximum request example is for a standard claim - so including it in our example adds confusion.

The side effect of including federalActivation on a standard claim request was that the zip code validation was treated like a BDD claim. For BDD claims, zip codes must be valid. So once we started mapping federalActivation data to the Form526 docker container, we were seeing errors using our example request.

Reply:
yeah, we're still supporting them as part of the request object, just not _that particular request example_. That being said, we don't have anything removing them if it were to _not be a BDD claim_, which is also concerning. Meaning folks can still send those dates with the request and we'll fail while submitting to the EVSS Docker Container with no noted documentation on the need to **_not provide them_** for a standard claim. 

Think we want to either add a note or some form of indicator on how we expect BDD vs Standard to receive / not receive the fedActivation dates. 

cc: @klwatanabe

Reply:
ignore, sounds like we're good with still sending with a BDD as long as it's valid.... and if it's submitted as invalid we're good with leaving as an errored status. so still capable of being sent in the request, and will just be validated by the docker container and report the error back if found.

Reply:
Understood. Is there a reason why we want it in the production swagger, but not in the dev swagger?

Reply:
@tycol7 The diff between dev and prod swagger files is a good catch. I must have made a mistake during a merge conflict. Should be fixed now.